test/iron-location.html

```diff
@@ -164,6 +167,76 @@
         assertHaveSameUrls(urlElem, otherUrlElem);
       });
     });
+    suite('when intercepting links', function() {
+
+      /**
+       * Returns whether an iron-location element with the given urlSpaceRegex
+       * if any would prevent the default behavior when the given anchor element
+       * is clicked.
+       */
+      function clickWouldBeCaptured(anchor, urlSpaceRegex) {
```

Comment:
Going back and re-reading the function as `clickDoesBeCaptured` is making me laugh.